src/ontology/mondo-edit.obo

```diff
@@ -437695,6 +437701,15 @@ intersection_of: has_modifier MONDO:0700005 ! idiopathic
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3169 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700060
+name: leukemia, acute, X-linked
+xref: OMIM:308960 {source="MONDO:equivalentTo"}
```

Comment:
can you please add a text def, following this pattern:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/x_linked.yaml

The def would be:
X-linked form of acute leukemia.
dbxref: MONDO:patterns/x_linked